docs/requirements.txt

```diff
@@ -4,7 +4,7 @@ funsor
 ipython
 jax
 jaxlib
-jaxns==2.4.8
+jaxns>=2.6.2
```

Comment:
could you pin the version instead? i dont think the version number reflects the stability of the library. We have many breakages in the past.

Reply:
I squashed everything in a new commit